pkg/output/plain.go

```diff
@@ -49,13 +48,12 @@ func (p *PlainPrinter) Print(_ context.Context, r *detectors.ResultWithMetadata)
 
 	if out.Verified {
 		boldGreenPrinter.Print("✅ Found verified result 🐷🔑\n")
-	} else if out.VerificationError != nil {
-		printer = yellowPrinter
-		boldYellowPrinter.Print("⚠️ Found result - unable to verify due to error 🐷🔑❗️\n")
-		printer.Printf("Verification Error: %s\n", out.VerificationError)
 	} else {
 		printer = whitePrinter
 		boldWhitePrinter.Print("Found unverified result 🐷🔑❓\n")
+		if out.VerificationError != nil {
+			yellowPrinter.Printf("Connection issue: %s\n", out.VerificationError)
```

Comment:
I'm trying to keep it consistent with our paid product. They are primarily issues with connecting with a provider.

Reply:
afaiu, we surface a subset of verification errors in enterprise as "connection issues" and open source surfaces all possible issues.

Also, if we change the name here we should also change the name in the [json printer](https://github.com/trufflesecurity/trufflehog/blob/4a5fbf84178018da2d92aa4d0a236add497da27b/pkg/output/json.go#L43) for consistency, though I'm still voting for keeping "verification error".

It would be good to gather other opinions since this is user-facing.

Reply:
Yeah there's already a divergence in functionality unless we want to bring the categorization logic back into the OSS tool, which I don't think anyone would argue for.

Maybe another way to approach is with the question "Will OSS users find the message `Connection issue: Unexpected HTTP Response: 500 Internal Server Error` confusing?" Because that's something that could happen after this change. I can see arguments both ways.

Reply:
We can go with verification error for now. It would still be a net improvement.

Cody, the answer is yes, some users are confused by these errors as they currently are surfaced. We've had people ask in some of our community venues.